modules/importer/src/runner/common/walker/git.rs

```diff
@@ -58,10 +58,11 @@ impl Handler for () {
     }
 }
 
-pub struct GitWalker<H, T>
+pub struct GitWalker<H, T, P>
```

Comment:
question: should we default `P=()` in that parameter?

Reply:
Technically possible. But there's a `new` function, which has that and is expected to be used instead of manually creating the struct.